website/src/util/RoadmapItems.ts

```diff
@@ -0,0 +1,114 @@
+/*****************************************************************/
+/**                                                              */
+/** GUIDE: ["Title", "Description", "Source", "Link to source"] **/
+/**                                                              */
+/*****************************************************************/
+
+// Product opportunities on our radar
+export const products_on_radar = [
+  ["Automatic caller ID lookup"],
+  ["Call recording"],
+  ["Call transcriptions"],
+  ["Call sentiment analysis for business"],
+  ["Call sentiment analysis for private"],
+  ["Call restriction"],
+  ["Call screening"],
+  ["Push to talk"],
+  ["Enterprise reputation scoring"],
+  ["Parental controls for kids"],
+  ["Family controls for elderly"],
+  ["Custom dial up tones"],
+  ["Noise cancellation"],
+  ["Automatic translation between numbers with different country codes"],
+  ["Burner number"],
+  ["Virtual landline"],
+  ["Voice recognition keys as password"],
+  ["SMS scheduler"],
+  ["Custom APN settings"],
+  ["Pre-integrated BSS"],
+  ["Pre-integrated customer portal"],
+  ["Virtual assistant to answer dial ups"],
+  ["Virtual assistant in calls "],
+  ["Corporate compliance on calls and texts"],
+  ["SMS Web client"],
+  ["Phone status"],
+  ["Live network updates"],
+  ["Subscription insights"],
+  ["Personalized carrier name"],
+  ["Time tracking"],
+  ["Travel roaming history"],
+  ["Secure OTP"],
+  ["Private Mobile Data Access"],
+]
+
+// Backlog (products we want to / are working to get on our platform)
+export const backlog = [
+  [
+    "Telavox",
+    "Telephony, PBX, messaging, meetings, and contact center in one simple platform",
+    "Telavox",
+    "https://telavox.com/",
+  ],
+  [
+    "VXT",
+    "Drive revenue and save time with integrated VoIP, video, and chat to make record keeping automatic",
+    "VXT",
+    "https://www.vxt.co.nz/",
+  ],
+  [
+    "Enghouse DMS",
+    "Sends the right handset settings to newly ported subscriptions",
+    "Enghouse",
+    "https://www.enghouse.com/",
+  ],
+  [
+    "Churn prevention",
+    "Identify what subscriptions are at risk of churning",
+    "Tech Batallion",
+    "https://thetechbattalion.com/",
+  ],
+  [
+    "Web Filtering Safe Surf",
+    "Make sure your not able to enter harmful websites",
+    "WG2",
```

Comment:
We aren't exactly consistent ourselves, but I think we should do a search-and-replace here: `/s/WG2/wgtwo`
```suggestion
    "wgtwo",
```